weaver/train.py

```diff
@@ -147,6 +149,8 @@
                     help='copy input files to the current dir (can help to speed up dataloading when running over remote files, e.g., from EOS)')
 parser.add_argument('--log', type=str, default='',
```

Comment:
```suggestion
parser.add_argument('--log-file', type=str, dest='log', default='',
```

Reply:
I think we can simply rename it? `argparse` allows partial matching, so then one can use either `--log` or `--log-file` (or any sub-str like `--log-f`). Is this sufficient to avoid the clash with `torchrun`?